src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeAlleleCounts.java

```diff
@@ -371,13 +371,12 @@ public String toUnphasedGenotypeString() {
             return "";
         }
         final StringBuilder sb = new StringBuilder(distinctAlleleCount * 3);
-        for (int i = 0; i < distinctAlleleCount; i += 2) {
+        for (int i = 0; i < distinctAlleleCount * 2; i += 2) {
```

Comment:
Can you avoid the multiplication * 2 at each iteration? Don't know if the VM will optimize this out. I wonder whether you could compare against sortedAlleleCounts.length but need to see the rest of the code to confirm.... too lazy 

If you change this loop consider to skip the +2 increment on i and simple increase as it is need to access the index and count:

```
for (int i = 0; i < stop; ) {
    int index = sortedAlleleCounts[i++];
    int count = sortedAlleleCounts[i++];
    ....
}
```